src/plugin/relativeTime/index.js

```diff
@@ -1,11 +1,13 @@
 import * as C from '../../constant'
 
 export default (o, c, d) => {
+  o = o || {}
   const proto = c.prototype
   d.en.relativeTime = {
     future: 'in %s',
     past: '%s ago',
     s: 'a few seconds',
+    ss: '%d seconds',
```

Comment:
seems we don't need this key.

Reply:
`ss` is built-in supported by `moment`, and if we don't set it here, if user passes a custom `ss` threshold, its default format would be `undefined`.

Reply:
we can leave this to our user, I think. Same reason, we can't update the English locale only.